src/PIL/Image.py

```diff
@@ -2446,6 +2446,23 @@ def save(self, fp, format=None, **params):
         if open_fp:
             fp.close()
 
+    def _save_all_progress(
```

Comment:
What's with "save all"? This method just passes data into another function (supplied externally).

If anything, `_report_progress` would make much more sense here. (Or even just `_progress`)

Reply:
It indicates that the method is used in relation to `im.save(out, save_all=True)`

Reply:
…Ah. Alright, it does make sense in that context.

Though it would still make more sense to name it something like `_frame_progress`, I'd say; it's more generic and there's not really anything in the method itself that makes it necessarily about saving only.